Exercise_1.py

```diff
@@ -1,19 +1,27 @@
+# O(N) time | O(1) Space
```

Comment:
Also define what 'N' is 
Space Complexity : O(n) where n is the size of the stack